### DISCUSSION TITLE
"Realtime" Safer Place AMA

### DISCUSSION THREAD

**Discussion by VoyTechnology** [SaferPlace/saferplace#143] [General]
__TL;DR;__ So much to do, we hope to launch by the end of the year, contributions welcome!

Behind the scenes for a while we have been working on a new version of Safer Place. This time focused on crowdsourced incident reports. We still have a long way to go, but I hope this platform will provide the community a place to document the issues happening in their community, issues which you would feel helpless to report otherwise, but with a power of the community, can highlight problems. The goal is to make a quick and safe way to report and view safety incidents, with very strong focus on user privacy.

## Why

While the development of the crowd sourced model for this project started back in 2022, during the summer of 2023 we have seen a heavy increase in crime in Dublin. Whether Dublin is turning worse by the year, or we are just reporting it more as part of the media fixation on the issues, it's clear that the feeling of hopelessness is growing. While I doubt this platform will solve the issue, my hope is that given an easy and quick way of reporting incidents, we can aggregate the data to present a cohesive view of dangerous areas, both to us users, so we can avoid them, but also make it publicly available that the public officials can see the areas they need to improve.

## When to Report

Of course, if someone's life is in danger, report it right away to the Gardai (and then on SaferPlace). This platform is a voice for those who thing "ah sure they won't do anything anyways". By making it simple, I hope that those things can be reported, and we have a clearer picture about the incidents around.

## What happens after the report

To prevent abuse, each report will be reviewed by a member of the review team. This team currently doesn't exist, but the role of the reviewer would be to look at the report, and screen it for racial biases (descriptions), ensure the data is valid (not spam), and remove duplicates. If the incident has a potential to affect you (eg. you are in the area), it will alert you of the incident. 

After that the incident will be visible to everybody on the map for a period of time.

In the future, the plan in to also provide heat maps, so we can see where the areas are with most reports.

## Privacy

We don't want your data. We are not going to log where you are when you are looking around at the map. Your phone will just get the blocks around you, with 1.11km precision, but it will try and randomise the blocks. This data will never be logged and associated with you.

Of course, when you make a report, we will get your precise location (so we know where it happened), but you will be able to change the precise location in case the GPS on your phone is off, or you have moved since the incident happened.

We also strive to be fully open source, and transparent with the operation costs. We will have a governance structure in place to handle incoming requests and concerns, but that will be more worked on once we have something actually worth deploying.

## Where we are now

The project at the moment can be fully ran locally, with the instructions in the README, even some of the things I mentioned are not fully completed.

## The Future

There is still soooo much to do. https://github.com/SaferPlace/saferplace/milestones provides the high level overview over the current plans, but if you want to dig down (or help 😉), the Github Project has all the tasks we want to do: https://github.com/orgs/SaferPlace/projects/2. The plan is to have a full launch by the end of the year, with public testing in between.

## More questions

I hope this answered some questions, but feel free to ask more.
